modules/guides/pages/creating-data.adoc

```diff
@@ -0,0 +1,265 @@
+= Creating Data
+:description: How to create documents in Couchbase.
+:page-pagination: full 
+:page-topic-type: guide
+:page-toclevels: 2
+:tabs:
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+include::partial$example-attributes.adoc[]
+
+[abstract]
+{description}
+
+Couchbase stores its xref:learn:data/data.adoc[data] as either JSON or Binary _documents_ with unique IDs (primary keys) by which they can be located.
+
+Documents are organized and grouped in _buckets_ using xref:tutorials:buckets-scopes-and-collections.adoc[scopes and collections]. 
+If a scope or collection is not provided when creating a document, Couchbase will automatically store data in the `_default` collection or `_default` scope.
+
+TIP: We highly recommend that you provide user-defined scopes and collections to logically group your data.
```

Comment:
I would get rid of this, as it really belongs in an explanatory guide to data modelling, rather than this how-to guide. Instead, include links to the client tools and a warning about altering your data.

```suggestion

include::partial$clients.adoc[tag=refs]

include::partial$data-warning.adoc[]

```